feature/survey/src/main/java/io/github/droidkaigi/confsched2019/survey/ui/SessionSurveyFragment.kt

```diff
@@ -240,6 +242,9 @@ class SessionSurveyFragment : DaggerFragment() {
             override fun isViewFromObject(view: View, `object`: Any): Boolean = view == `object`
 
             override fun getCount(): Int = enumValues<SurveyItem>().size
+
+            // views should be recreated on PagerAdapter#notifyDataSetChanged
+            override fun getItemPosition(`object`: Any): Int = POSITION_NONE
```

Comment:
I think this is not the best solution because views are not recycled, but `PagerAdapter` seems to have no good way to recycle Views.